src/Jobs/Collection.php

```diff
@@ -133,6 +133,18 @@ public function tagged(string $tag, int $offset = 0, int $limit = 25): array
         return $response['jobs'];
     }
 
+    /**
+     * Fetches a list of tracked jobs.
+     *
+     * @return array
+     */
+    public function tracked(): array
+    {
+        $tracked = $this->client->call('tracked') ?? [];
```

Comment:
```suggestion
        $tracked = $this->client->call('tracked') ?? '';
```
Інакше при 
`json_decode([], true)`
буде
_json_decode() expects parameter 1 to be string, array given_

Reply:
Дякую, так, буде бага